sdk/clientcore/core/src/main/java/io/clientcore/core/http/pipeline/HttpInstrumentationPolicy.java

```diff
@@ -644,4 +644,9 @@ public void close() throws IOException {
             originalBody.close();
         }
     }
+
+    @Override
+    public HttpPipelineOrder getOrder() {
```

Comment:
Let's make this method `final` too like the HttpCredentialPolicy.

Reply:
This class is `final`, so adding `final` to a method results in Checkstyle or Spotbugs raising a warning. We can do this but will need to add the corresponding suppression.